dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -851,20 +859,63 @@ sources:
-                  min_value: 1
-                  max_value: 10000
-                  additional_select_columns: *select-columns
+                  name: iasworld_dweldat_sfla_between_1_and_10000
+                  min_value: |
+                    CASE
+                      WHEN class = '202' THEN 1
+                      WHEN class = '203' THEN 1000
+                      WHEN class = '204' THEN 1801
+                      WHEN class = '205' THEN 1
+                      WHEN class = '206' THEN 2201
+                      WHEN class = '207' THEN 1
+                      WHEN class = '208' THEN 3801
+                      WHEN class = '209' THEN 5000
+                      WHEN class = '210' THEN 1
+                      WHEN class = '211' THEN 1
+                      WHEN class = '212' THEN 1
+                      WHEN class = '234' THEN 1
+                      WHEN class = '278' THEN 2001
+                      WHEN class = '295' THEN 1
+                      ELSE 1
+                    END
+                  max_value: |
+                    CASE
+                      WHEN class = '202' THEN 999
+                      WHEN class = '203' THEN 1800
+                      WHEN class = '204' THEN 10000
+                      WHEN class = '205' THEN 2200
+                      WHEN class = '206' THEN 4999
+                      WHEN class = '207' THEN 2000
+                      WHEN class = '208' THEN 4999
+                      WHEN class = '209' THEN 20000
+                      WHEN class = '210' THEN 10000
+                      WHEN class = '211' THEN 20000
+                      WHEN class = '212' THEN 20000
+                      WHEN class = '234' THEN 10000
+                      WHEN class = '278' THEN 3800
+                      WHEN class = '295' THEN 10000
+                      ELSE 20000
+                    END
+                  additional_select_columns:
+                    - taxyr
+                    - parid
+                    - card
+                    - class
+                    - who
+                    - wen
                   config:
                     where: |
                       CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
-                      AND class NOT IN ('208', '209', '211', '212', '236', '297')
+                      AND class IN (
+                        '202', '203', '204', '205', '206', '207', '208', '209',
+                        '210', '211', '212', '234', '278', '295'
+                      )
                       AND cur = 'Y'
                       AND deactivat IS NULL
                   meta:
                     description: >
-                      sfla (Building Square Footage) should be between 1 and 10000
-                      for single-family properties (excl. 208 or 209)
+                      sfla (Building Square Footage) should match the class
+                      definition for class 2 regression properties
```

Comment:
Does this seem reasonable to you? It relies on an assumption that the square foot thresholds set in our [class definitions](https://github.com/ccao-data/wiki/blob/master/Data/Class-Definitions.pdf) are hard and fast rules, but I don't have full confidence in that.

Reply:
This makes senses for now, as they _are_ technically supposed to be hard definitions. I commented the ones that need specific changes.